guide/C/ch_txns.xml

```diff
@@ -349,10 +349,10 @@
         <note>
           <para>Note: An account register displays a transaction when that
           transaction has a split assigned to the account. If a given transaction
-          has more than one split assigned to a single account, then in Basic and
-          Auto-Split view modes that transaction will appear in the register one
+          has more than one split assigned to the register account, then in Basic and
```

Comment:
I agree on this one. "the register account" requires me to read the sentence more than once to grasp what is meant. View modes to views is fine for me.